sreftml/utilities.py

```diff
@@ -10,6 +10,13 @@
 from sklearn.linear_model import LinearRegression
 
 
+class NullModel:
+    params = [np.nan, np.nan]
```

Comment:
I don't think class variables are necessary.

Reply:
Yes, I'll correct it.